src/containers/find-offer/filter-bar-menu/FilterBarMenu.tsx

```diff
@@ -0,0 +1,94 @@
+import { FC, useState, ChangeEvent } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import Box from '@mui/material/Box'
+
+import useBreakpoints from '~/hooks/use-breakpoints'
+
+import AppContentSwitcher from '~/components/app-content-switcher/AppContentSwitcher'
+import AppSelect from '~/components/app-select/AppSelect'
+import ViewSwitcher from '~/components/view-switcher/ViewSwitcher'
+import FiltersTitle from '~/components/filters-title/FiltersTitle'
+
+import { styles } from '~/containers/find-offer/filter-bar-menu/FilterBarMenu.styles'
+import { sortByFields, initialBarMenuFilters } from '~/containers/find-offer/filter-bar-menu/FilterBarMenu.constants'
+
+import { BarMenuFilters, CardsViewTypes } from '~/types'
+
+
+interface FilterBarMenuProps {
+  chosenFiltersQty:number,
+  openFilters: () => void,
+  getFilters: (filters: BarMenuFilters) => void,
+  handleOffersView:(view: CardsViewTypes) => void,
+  offersView: CardsViewTypes
+}
+
+const FilterBarMenu: FC<FilterBarMenuProps> = ({ chosenFiltersQty, openFilters, getFilters, handleOffersView, offersView }) => {
+  const [barMenuFilters, setBarMenuFilters] = useState<BarMenuFilters>(initialBarMenuFilters)

```

Comment:
i dont think we need state here, we will manage all filters outside, just pass them as prop

Reply:
but maybe passing all filters will be unnecessary for this component, so it will be better to have filters only it needs locally and on changing just move them outside and setting them in state of others filters

Reply:
if you don't pass all filters you would not know how many active filters you have to chow on the icon near "Filter"

Reply:
your filters and filters from sidebar will be in one object

Reply:
this quantity we can count outside and just pass a number of it, there is such prop "chosenFiltersQty"

Reply:
yep but your filters affect on cards show also as other filters I don't see the point of why they have to be separate

Reply:
i think your way is good, but how do you think is it good that this container will be getting from props all filters, that it needs and that does not need

Reply:
for me, it`s a better solution than one more state which can be avoided

Reply:
okey, thx